frontends/web/src/routes/device/bitbox02/createbackup.tsx

```diff
@@ -35,15 +35,14 @@ export const Create = ({ deviceID }: TProps) => {
   const createBackup = () => {
     setCreatingBackup(true);
     createBackupAPI(deviceID)
-      .then(() => {
+      .then((result) => {
         setCreatingBackup(false);
         setDisabled(false);
+        if (!result.success) {
+          alertUser(t('backup.create.fail'));
+        }
       })
-      .catch(() => {
-        setCreatingBackup(false);
-        setDisabled(false);
-        alertUser(t('backup.create.fail'));
-      });
+      .catch(console.error);
```

Comment:
Just a general question for all the changes, what happens if the API call resolves in an error (in this case the createBackupAPI)? Will it catch and `console.error` only? So the user won't be notified? Orr...?

Reply:
understood, thanks.